typings/test-typings.ts

```diff
@@ -43,3 +43,5 @@ console.log(getMcData.supportedVersions.pc)
 console.log(getMcData.schemas.blocks)
 
 console.log(getMcData('1.12').language['options.sensitivity.max'])
+console.log(getMcData.getRenamedData('blocks', 'short_grass', '1.20.3', '1.8.8') === 'tallgrass') // test string comparison
+console.log(getMcData.getRenamedData('blocks', ['standing_sign', 'grass'], '1.8.8', '1.16'))
```

Comment:
will be [ 'oak_sign', 'grass_block' ]

Reply:
This is an example of calling the API but not an example of usage. I don't think we should ever try to do that.

Clearly tall and short are not the same thing

Oak is a specific kind of wood

Grass and grass block may be different

Having tags and groups make sense to note similar blocks that should be treated similarly

Reply:
> Having tags and groups make sense to note similar blocks that should be treated similarly

It depends on the use case. As I mentioned earlier there are a lot of use cases when you want to track block renames across the versions (such as world upgrades and so on).

Here `short_grass` and `tallgrass` are the same thing actually (dont' let 1.8.8 naming fool you)